bootstrap/main.go

```diff
@@ -64,30 +68,39 @@ func mainImpl() {
 	}
 
 	if !kubectlClient.IsPresent() {
+		sendError("Missing kubectl binary, please install before continuing.", opts)
 		exitWithCapture("Could not find kubectl in PATH, please install it: https://kubernetes.io/docs/tasks/tools/install-kubectl/\n")
 	}
 
 	agentK8sURL, err := text.ResolveString(agentK8sURLTemplate, opts)
 	if err != nil {
+		sendError("Error occurred during installing agents.", opts)
 		log.Fatal("invalid URL template:", err)
 	}
 	wcOrgLookupURL, err := text.ResolveString(weavecloud.DefaultWCOrgLookupURLTemplate, opts)
 	if err != nil {
+		sendError("Error occurred during installing agents.", opts)
 		log.Fatal("invalid URL template:", err)
 	}
 
 	// Restore stdin, making fd 0 point at the terminal
 	if err := syscall.Dup2(1, 0); err != nil {
+		sendError("Could not restore stdin", opts)
 		exitWithCapture("Could not restore stdin\n", err)
```

Comment:
The repeated pairing of `sendError` and `exitWithCapture`, with near-identical-content, should be replaced with a function that does both. And I see no reason for the text to differ.

Reply:
My reasoning behind it, as we have in some places instructions that say `...run kubectl version....`, etc., and I think displaying that in the UI is not the right place but the CLI is. 

The content of the `sendError` msgs might change, as @ngehani might have different suggestions.

But if it's okay with everyone, I am happy to just call `sendError` in `exitWithCapture`.

Reply:
It's ok to say "run kubectl version" in the UI. As for the content of the error messages changing - I would expect any changes to affect both places. There's certainly a lot to be said for showing the same error in the UI as in the CLI.

Reply:
Ok, then will return the same error to CLI and UI, sgtm. 👍